raft/raftpb/raft.proto

```diff
@@ -13,12 +14,16 @@ option (gogoproto.goproto_unrecognized_all) = false;
 option (gogoproto.goproto_sizecache_all) = false;
 
 enum EntryType {
+	option (versionpb.etcd_version_enum) = "3.0";
+
 	EntryNormal       = 0;
-	EntryConfChange   = 1; // corresponds to pb.ConfChange
-	EntryConfChangeV2 = 2; // corresponds to pb.ConfChangeV2
+	EntryConfChange   = 1 ; // corresponds to pb.ConfChange
```

Comment:
Is this empty space a mistake or does it serve a purpose?

Reply:
Nope, just a mistake that I didn't noticed.
```suggestion
	EntryConfChange   = 1; // corresponds to pb.ConfChange
```